components/brave_extension/extension/brave_extension/content_element_picker.ts

```diff
@@ -274,6 +274,7 @@ const launchElementPicker = () => {
     'border: 0',
     'border-radius: 0',
     'box-shadow: none',
+    'color-scheme: light only',
```

Comment:
This change broke Element Picker everywhere when using the native Auto Dark modes
- Brave://flags -> Auto Dark Mode for Web Contents -> Any mode
- or the temporary way to test it in Devtools -> Rendering -> Enable Automatic Dark mode

For the way native Auto dark mode works, where it doesn't try to change/override an already made page's dark theme, and only works if there is no `color-scheme: dark` present or applied to a page, it seems that the 'light only' is forcing the auto dark mode to get enabled and then cause the issues with the non-transparent background **on every page**.

I tested on Stable and as soon as I add the same light only CSS property, it causes the issues appears, but changing it to auto or light doesn't seem to affect either Auto dark mode or Dark Reader extension.

Reply:
CSS :sweat: 

`color-scheme: auto` seems to work better with the auto dark mode, I'll push a new PR for that

Reply:
### update

even better (https://www.w3.org/TR/css-color-adjust-1/#color-scheme-effect, emphasis mine):

> In order to preserve expected color contrasts, in the case of embedded documents typically rendered over a transparent [canvas](https://www.w3.org/TR/CSS2/intro.html#canvas) (such as provided via an HTML [iframe](https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-iframe-element) element), **if the [used color scheme](https://www.w3.org/TR/css-color-adjust-1/#used-color-scheme) of the element and the used color scheme of the embedded document’s root element do not match, then the UA must use an opaque canvas of the [Canvas](https://www.w3.org/TR/css-color-4/#valdef-system-color-canvas) color appropriate to the embedded document’s used color scheme instead of a transparent canvas**. This rule does not apply to documents embedded via elements intended for graphics (such as [img](https://html.spec.whatwg.org/multipage/embedded-content.html#the-img-element) elements embedding an SVG document).

So if the iframe and inner document specify that they're compatible with both light and dark color schemes (`color-scheme: light dark`), then it should work as expected.